packages/hide/test/index.js

```diff
@@ -33,11 +15,8 @@ describe('@rebass/hide', () => {
         xlarge
       />
     )
-    console.log('json', json)
+
     expect(json).toMatchSnapshot()
-    expect(json).toHaveStyleRule('display', 'none', {
-      // media: 'screen and (min-width:40em)'
```

Comment:
I think this was maybe a bug with jest-styled-components, would be good to have tests for the media query aspect of this component

Reply:
The bug still appears. There is not media queries with any combination. Reverted comment. Not related to this PR.